tools/shared/src/main/scala/org/scalajs/core/tools/jsdep/FlatJSDependency.scala

```diff
@@ -7,11 +7,15 @@ import org.scalajs.core.ir.Trees.isValidIdentifier
  */
 final class FlatJSDependency(
     val origin: Origin,
-    val resourceName: String,
+    @deprecatedName('resourceName)
+    val relPath: String,
```

Comment:
Oh... actually it is called `resourceName` in `JSDependency`. Could you just add a comment there that it may also be a partial relative path?